docs/functions.rst

```diff
@@ -150,6 +150,82 @@ Example:
         - |
           echo "Example"
 
+KubeTaskParallel
```

Comment:
I think maybe we should the name of the function in next community call? It's good right now, but by reading the name it looks like in this function we are doing things parallelly that kubetask does sequentially, which doesn't seem to be the case right?
I think it doesn't necessarily have to have KubeTask in it's name right? Maybe `MultiContainerUpload` or something in that line?

Reply:
I was trying to avoid the "purpose" style names like "dump", "upload" or "export". While this function is used for that purpose it's deliberately made generic.

Reply:
ok, let's see what others think @pavannd1 @PrasadG193 @mlavi .

Reply:
Somehow `KubeTaskParallel` doesn't sound good to me. I agree that it would be better if we avoid `KubeTask*` prefix for the new function.

Reply:
We can use `MultiContainerPod` or `MultiContainerRun`, but we already use `KubeTaslk` instead of `Pod` in the other function.

Reply:
I like `MultiContainerRun`. Do we need to change the arguments name as well? that are currently named background and ouptut I think.

Reply:
Naming is hard. Okay with `MultiContainerRun` or maybe `MultiContainerJob`

Reply:
I don't have better names than `background` and `output` for containers.

Reply:
I was thinking about the use of the named pipe - one of the containers pushes data while the other one receives it.
Here are some options I could think of:
- `input` and `output`
- `producer` and `consumer`
- `reader` and `writer`

Reply:
> I was thinking about the use of the named pipe - one of the containers pushes data while the other one receives it.

That is not a part of the function itself, but a part of the command. The only practical difference in the implementation of those containers is that only one of them sends its ouput into the function output. Otherwise you can make the output container wirte data to a file and you can make the background container read from the file.